### PR TITLE
add nebula-console package action

### PR DESCRIPTION
**Pull request by czpmango**
## What type of PR is this?
- [ ] bug
- [x] feature
- [ ] enhancement

#### Description:
Package nebula-console into an RPM /deb package through action.

## Checklist:
Tests:
- [ ] Unit test(positive and negative cases)
- [ ] Function test
- [ ] Performance test
- [ ] N/A

Affects:
- [x] Documentation affected (Please add the label if documentation needs to be modified.)
- [ ] Incompatibility (If it breaks the compatibility, please describe it and add the label.）
- [ ] If it's needed to cherry-pick (If cherry-pick to some branches is required, please label the destination version(s).)
- [ ] Performance impacted: Consumes more CPU/Memory

## Release notes:
 - The RPM/DEB package will contain the nebula-console binary

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -33,6 +33,9 @@ jobs:
         with:
           run: sh -c "find . -mindepth 1 -delete"
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+      - name: compile nebula-console
```
Comment (yixinglu) [inline .github/workflows/nightly.yml:37]:
```yaml
- uses: actions/checkout
  with:
     repository: vesoft-inc/nebula-console
     ref: ${{ github.ref }}
     directory: nebula-console
- run: |
      make && mv nebula-console ../build/bin/nebula
   working-directory: nebula-console
```

Comment (yixinglu) [reply .github/workflows/nightly.yml:37]: maybe it's better to do it by cmake

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -33,6 +33,9 @@ jobs:
         with:
           run: sh -c "find . -mindepth 1 -delete"
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
```
Comment (yixinglu) [inline .github/workflows/nightly.yml:36]: specify golang version

File: .github/workflows/nightly.yml (hunk 1 above)
Comment (czpmango) [reply .github/workflows/nightly.yml:37]: ACK.

File: .github/workflows/nightly.yml (hunk 2 above)
Comment (czpmango) [reply .github/workflows/nightly.yml:36]: ACK.

**Comment by czpmango** [comment]
Closed because A better implementation is here https://github.com/vesoft-inc/nebula/pull/3905

**Comment by HarrisChu** [comment]
we should prevent thrift version are different between nebula graph and nebula console.

**Pull request closed without merging**